tests/plugin_http_api_test.py

```diff
@@ -106,12 +106,29 @@ def test_ChainApi(self) :
         allFeatureDigests = [d['feature_digest'] for d in allProtocolFeatures]
         ACT_FEATURE_DEFAULT_LIMIT = 10
 
+        prot_feat_cmd = cmd_base + "get_activated_protocol_features"
+        default_cmd = prot_feat_cmd
+        def get_activated_protocol_features():
+            ret_json = Utils.runCmdReturnJson(prot_feat_cmd)
+            activated_protocol_features = ret_json["activated_protocol_features"]
+            self.assertEqual(type(activated_protocol_features), list)
+            return activated_protocol_features
+
+        initial_prot_feat = get_activated_protocol_features()
+
+        # wait till all the protocol features are activated in the blocks
+        pre_block = self.nodeos.getBlockNum()
+        later_block = pre_block + 2
+        self.nodeos.waitForBlock(later_block, reportInterval=1)
```

Comment:
Why not modify preactivateAllBuiltinProtocolFeature on Node.py or maybe add some comment to it? i am not sure if this is is going to be used again in the future, but i think i was expecting that after returning from that method (on line 84) the nodeos node is expected to be on a state that has those activated and don't have to remember to do this wait separately